app/modules/entities/components/editor/lib/suggestions/property_values_shortlist.js

```diff
@@ -1,18 +1,13 @@
-import preq from '#lib/preq'
+import { API } from '#app/api/api'
 import { pluralize } from '#entities/lib/types/entities_types'
+import assert_ from '#lib/assert_types'
+import preq from '#lib/preq'
+
+export const allowedValuesPerTypePerProperty = await preq.get(API.data.propertyValues).then(({ values }) => values)
```

Comment:
Wow, that's quite something to read. I guess that happens often to old fools who see the world (and webpack) moving around them.

Reply:
Right?!? ^^ It has more to do with the [ES modules having been standardized as async](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#top_level_await), than webpack which just complies to that standard